### DISCUSSION TITLE
How to Slack with Us

### DISCUSSION THREAD

**Discussion by kyleaton** [great-expectations/great_expectations#4948] [General]
**Welcome and thanks for joining the Great Expectations Slack Community!**

https://greatexpectations.io/slack

Our community of data practitioners is growing a lot and has become an invaluable resource for everyone using Great Expectations. So to help us keep that rolling we created our “How to Slack with Us” below:

We have [Slack Guidelines](https://github.com/great-expectations/great_expectations/blob/develop/SLACK_GUIDELINES.md). Please read them out as they highlight how to ask for help, channels, and code of conduct. Here are some highlights:

- Be supportive - This is an open source community. We’re helping each other build something useful and it’s free! We all rely on the community to jump in and help.
- Use threads to keep conversations organized.
- Avoid posting large blocks of code in the main channels (threads can be used instead).
- Avoid posting sensitive information.
- Avoid using DMs for support - Posting in public enables the community to learn from your question and allows everyone to respond.
- Make sure to post questions in the appropriate channels!
- Follow this guide on how to write [effective questions](https://discuss.greatexpectations.io/t/how-to-ask-a-support-question/749). We’ll be able to answer your questions faster and more efficiently if you follow these rules when posting!
- If you're unsure about the [Promoting Products and Services](https://github.com/great-expectations/great_expectations/blob/develop/SLACK_GUIDELINES.md#promoting-products-or-services) rule. Just ask [@Kyle Eaton](https://greatexpectationstalk.slack.com/team/UJNF5LH7D)

**Want to be a contributor :coffee::beer:?**

1. Check out our [How to contribute page](https://github.com/great-expectations/great_expectations/blob/develop/CONTRIBUTING.md) that highlights how and where you can jump in. Also jump into [#contributors-contributing](https://greatexpectationstalk.slack.com/archives/CV828B2UX) to talk to prospective and current contributors.
2. Have you already implemented your own features to your GE and want to share it with the community? DM [@Kyle Eaton](https://greatexpectationstalk.slack.com/team/UJNF5LH7D)

**[Read our documentation](http://docs.greatexpectations.io/) if you haven’t already.**
Have a question?

- Check out https://github.com/great-expectations/great_expectations/discussions :discuss: to see if your question has been answered or is being discussed.
- Or ask for some help in  [#support](https://greatexpectationstalk.slack.com/archives/CUTCNHN82) or [#beginners](https://greatexpectationstalk.slack.com/archives/CUUMYDJAX)

**Update your profile (these are optional but help the community)**

- Enter your full name (this community will work best if we are all real humans), but feel free to add a custom “display name”
- Add a headline in the “what I do” box (can be “title @ company” or anything that helps people know who they are chatting with).
- Add a profile photo
- Intro yourself in [#introductions](https://greatexpectationstalk.slack.com/archives/CKBLL4HT7)

**Planning or interested in writing a blog or doing a talk including GE?**

- Feel free to DM [@Kyle Eaton](https://greatexpectationstalk.slack.com/team/UJNF5LH7D), we would love to help! The same goes for if you just need a logo.

**We are looking to grow our “Trusted by” section.**

- If your team is happily using GE we would love to have the privilege to display your logo on our site with the other proud users of GE. DM Matthew Lundgren if you’re interested.

Have fun out there and enjoy!
Last but not least...  :star: us on GitHub! It really makes a difference for open source projects!

**Comment by guido-vries** [comment]
HI! The invite link is no longer working. Is the slack still active?

**Reply by kyleaton** [reply]
The link should now be live @ActivateConsulting! Thanks for commenting!
